R/dvhydrograph-render.R

```diff
@@ -89,6 +97,11 @@ createDVHydrographPlot <- function(reportObject){
   estimated3EdgesDf <- as.data.frame(estimated3Edges, stringsAsFactors = FALSE)
   estimated4EdgesDf <- as.data.frame(estimated4Edges, stringsAsFactors = FALSE)
   comparisonEdgesDf <- as.data.frame(comparisonEdges, stringsAsFactors = FALSE)
+  
+  #remove zeros from field measurements, if present:
+  if (excludeZeroNegativeFlag && !isEmptyOrBlank(fieldVisitMeasurements)) {
+    fieldVisitMeasurements <- removeZeroNegative(fieldVisitMeasurements)
+  }
```

Comment:
This actually removes the zeros from the data if excludeZeroNegativeFlag = true; otherwise that doesn't happen and the report doesn't render if excludeZeroNegativeFlag = true and the zeros and negatives are not removed.